### DISCUSSION TITLE
Wrong steam in New Caledonia play list

### DISCUSSION THREAD

**Discussion by The-MAD-Virus** [iptv-org/community#41] [Q&A] [answered]
Hi,

Can some one clean the New Caledonia play list ?
(https://iptv-org.github.io/iptv/countries/nc.m3u)

There is not even one New Caledonia channel.

**Comment by BellezaEmporium** [comment, marked as answer]
The nc.m3u playlist, right now, contains Internationally-streamed channels.

**Comment by loucasal** [comment]
Please support iptv-org/iptv#8340 if that looks counterintuitive to you too.
